src/css/layout/ScrollbarLayout.css

```diff
@@ -67,10 +64,8 @@
 }
 
 .ScrollbarLayout/faceHorizontal:after {
-  bottom: var(--scrollbar-face-margin);
```

Comment:
Previously, top, bottom, left, right were set to align the thumb in the center of the scrollbar face element.
This is a bit hard to read and manage, and we can instead simplify this with flex layout.